src/redux/store.ts

```diff
@@ -17,19 +25,27 @@ const persistConfig = {
 
 // Setup Middlewares
 const sagaMiddleware = createSagaMiddleware();
-const multipleApplyMiddleware = applyMiddleware(sagaMiddleware);
+const middleware = [
+  ...getDefaultMiddleware({
+    thunk: false,
+    // This is needed because redux-persist will prompt an error.  Redux-persist is using default redux configuration and redux-starter-kit is waiting for a string.
+    serializableCheck: {
+      ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
+    },
+  }),
+  sagaMiddleware,
+];
 
 // Setup Reducers
 const persistedReducer = persistCombineReducers(persistConfig, reducers);
 
-// Setup Enhancers
-const enhancers =
-  Platform.OS === 'ios'
-    ? composeWithDevTools(multipleApplyMiddleware)
-    : multipleApplyMiddleware;
-
 // Create Store
-const store = createStore(persistedReducer, enhancers);
+const store = configureStore({
+  reducer: persistedReducer,
+  middleware,
+  devTools: process.env.NODE_ENV !== 'production',
```

Comment:
These two lines look odd.  You're making the devtools conditional, but also unconditionally adding them in the next line?

Note that you can pass DevTools options as the `devTools` argument, so you could do something like:

```js
const devTools = process.env.NODE_ENV !== "production" ? {realTime: true} : false;
```

Reply:
the last line was a mistake. I forgot there. Thanks to point it out.